src/appmixer/google/calendar/CreateEvent/component.json

```diff
@@ -5,8 +5,7 @@
     "auth": {
         "service": "appmixer:google",
         "scope": [
-            "https://www.googleapis.com/auth/calendar",
-            "https://www.googleapis.com/auth/calendar.readonly"
+            "https://www.googleapis.com/auth/calendar.events"
```

Comment:
`https://www.googleapis.com/auth/calendar.events` is restricted scope, requires verification 
![image](https://github.com/user-attachments/assets/4e57ca09-277c-4bba-b384-d304ea492c13)